centreon/tests/e2e/features/Contacts/01-contact-configuration/index.ts

```diff
@@ -0,0 +1,120 @@
+  }).as('getNavigationList');
+  cy.intercept({
+    method: 'GET',
+    url: '/centreon/include/common/userTimezone.php'
+  }).as('getTimeZone');
+});
+
+afterEach(() => {
+  cy.stopContainers();
+});
+
+Given('an admin user is logged in a Centreon server', () => {
+  cy.loginByTypeOfUser({
+    jsonName: 'admin',
+    loginViaApi: false
+  });
+});
+
+When('a contact is configured', () => {
+  cy.navigateTo({
+    page: 'Contacts / Users',
+    rootItemNumber: 3,
+    subMenu: 'Users'
+  });
+  cy.wait('@getTimeZone');
+  cy.getIframeBody().contains('a', 'Add').click();
+  cy.addOrUpdateContact(contacts.default);
+});
+
+When('the user updates some contact properties', () => {
+  cy.getIframeBody().contains(contacts.default.alias).click();
+  cy.addOrUpdateContact(contacts.contactForUpdate);
+});
+
+Then('these properties are updated', () => {
+  cy.getIframeBody().contains(contacts.contactForUpdate.alias).should('exist');
+  cy.getIframeBody().contains(contacts.contactForUpdate.alias).click();
+  cy.wait('@getTimeZone');
+  cy.waitForElementInIframe('#main-content', 'input[id="contact_alias"]');
+  cy.getIframeBody()
+    .find('input[id="contact_alias"]')
+    .should('have.value', contacts.contactForUpdate.alias);
+  cy.getIframeBody()
+    .find('input[id="contact_name"]')
+    .should('have.value', contacts.contactForUpdate.name);
+  cy.getIframeBody()
+    .find('input[id="contact_email"]')
+    .should('have.value', contacts.contactForUpdate.email);
+  cy.getIframeBody()
+    .find('input[id="contact_pager"]')
+    .should('have.value', contacts.contactForUpdate.pager);
+  cy.getIframeBody().find('#contact_template_id').should('have.value', '19');
+  cy.checkLegacyRadioButton(contacts.contactForUpdate.isNotificationsEnabled);
+});
+
+When('the user duplicates the configured contact', () => {
+  checkFirstContactFromListing();
+  cy.getIframeBody().find('select[name="o1"]').select('Duplicate');
+  cy.wait('@getTimeZone');
+  cy.exportConfig();
+});
+
+Then('a new contact is created with identical properties', () => {
+  cy.getIframeBody().contains(`${contacts.default.alias}_1`).should('exist');
+  cy.getIframeBody().contains(`${contacts.default.alias}_1`).click();
+  cy.waitForElementInIframe('#main-content', 'input[name="contact_alias"]');
+
+  cy.getIframeBody()
+    .find('input[name="contact_alias"]')
+    .should('have.value', `${contacts.default.alias}_1`);
+  cy.getIframeBody()
+    .find('input[name="contact_name"]')
+    .should('have.value', `${contacts.default.name}_1`);
+  cy.getIframeBody()
+    .find('input[name="contact_email"]')
+    .should('have.value', contacts.default.email);
+  cy.getIframeBody()
+    .find('input[name="contact_pager"]')
+    .should('have.value', contacts.default.pager);
+  cy.getIframeBody().find('#contact_template_id').should('have.value', '19');
+  cy.checkLegacyRadioButton(contacts.default.isNotificationsEnabled);
```

Comment:
I think this function does not exist in dev-24.10.x

Reply:
For some PRs, I have to merge other PRs and resolve conflicts for it to work, each branch is based on the changes I made on other branches and since they haven't merged yet the tests fail!
I'm doing this little by little!